### DISCUSSION TITLE
posts/linux_namespace_cgroup/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#49] [General]
# posts/linux_namespace_cgroup/

介绍Linux 命名空间**(Namespace)** Linux 命名空间是一种隔离机制，允许将全局系统资源划分为多个独立的、相互隔离的部分，使得在不同

http://localhost:1313/posts/linux_namespace_cgroup/

**Comment by binbin0325** [comment]
good
